src/mpid/ch4/src/ch4r_callbacks.h

```diff
@@ -35,5 +35,11 @@ int MPIDIG_ssend_ack_target_msg_cb(int handler_id, void *am_hdr, void *data,
 int MPIDIG_send_long_ack_target_msg_cb(int handler_id, void *am_hdr, void *data,
                                        MPI_Aint p_data_sz, int is_local, int is_async,
                                        MPIR_Request ** req);
+int MPIDIG_send_gpu_ipc_req_target_msg_cb(int handler_id, void *am_hdr, void *data,
+                                          MPI_Aint in_data_sz, int is_local, int is_async,
+                                          MPIR_Request ** req);
+int MPIDIG_send_gpu_ipc_ack_target_msg_cb(int handler_id, void *am_hdr, void *data,
+                                          MPI_Aint in_data_sz, int is_local, int is_async,
+                                          MPIR_Request ** req);
```

Comment:
Can we move them to `mpidig_am_send.h`?

Reply:
I am resolving these comments as I moved the code the a dedicated shmmod